edlib/src/edlib.tpp

```diff
@@ -352,16 +367,16 @@ extern "C" char* edlibAlignmentToCigar(const unsigned char* const alignment, con
  * Bit i of Peq[s * maxNumBlocks + b] is 1 if i-th symbol from block b of query equals symbol s, otherwise it is 0.
  * NOTICE: free returned array with delete[]!
  */
-static inline Word* buildPeq(const int alphabetLength,
-                             const unsigned char* const query,
-                             const int queryLength,
-                             const EqualityDefinition& equalityDefinition) {
+template <class AlphabetIdx>
+static inline edlib::internal::Word* edlib::internal::buildPeq(const int alphabetLength, const AlphabetIdx *query,
```

Comment:
Since the user has to include edlib.hpp (and edlib.tpp is included in it), if we put using edlib or using edlib::internal they will have access to the private implementation and again the conflicts we had before will appear. 
I can put all the code in the just one `namespace edlib{}` without any indentation as you said and wrap the declarations and definitions of the private functions into the nested `namespace internal{}` again without any indentation. Could we do that?

Reply:
Ah right, I keep forgetting this is all in header.

Ok, well this is also ok then, it is not so terrible.

Your suggestion -> I am not sure I got it correctly, nor am I sure if it can be done :), but I trust you to figure out what is best then -> whatever you think is the nicest way to go, let's do it.

Reply:
I'll do what I think in the next PR. After that, you can decide better.